katas/langtonant/israel/ant.test.js

```diff
@@ -0,0 +1,66 @@
+import { ant } from './ant';
+
+// test newDirection
+test('Method test newDirection for ant class', () => {
+    const p = [2, 2];
+    let ant1 =  new ant(p, 'up');
+    const colors = ['black', 'white'];
+    const directions = ['up', 'down', 'left', 'right'];
+    // for black color
+    for (const d of directions) {
```

Comment:
This `for` block might not be actually needed. Normally, unit tests are expected to hardcode values for each test, in other words, your test could be simply:
```
expect(ant1.newDirection('up', colors[0])).toBe('left');
expect(ant1.newDirection('down', colors[0])).toBe('right');
expect(ant1.newDirection('left', colors[0])).toBe('down');
expect(ant1.newDirection('right', colors[0])).toBe('up');
```